### PR TITLE
feat(bedrock): New embedding model config

### LINKED ISSUE #657: (bedrock): Knowledge base BedrockEmbeddingModelConfiguration
### Describe the feature

Updates to Knowledge Base in cdk v2.154.1

```
[~] resource AWS::Bedrock::KnowledgeBase
│    │ └ types
│    │    ├[+] type BedrockEmbeddingModelConfiguration
│    │    │ ├  documentation: The vector configuration details for the Bedrock embeddings model.
│    │    │ │  name: BedrockEmbeddingModelConfiguration
│    │    │ └ properties
│    │    │    └Dimensions: integer
│    │    ├[+] type EmbeddingModelConfiguration
│    │    │ ├  documentation: The configuration details for the embeddings model.
│    │    │ │  name: EmbeddingModelConfiguration
│    │    │ └ properties
│    │    │    └BedrockEmbeddingModelConfiguration: BedrockEmbeddingModelConfiguration
│    │    └[~] type VectorKnowledgeBaseConfiguration
│    │      └ properties
│    │         └[+] EmbeddingModelConfiguration: EmbeddingModelConfiguration
```

### Use Case

missing features

### Proposed Solution

_No response_

### Other Information

_No response_

### Acknowledgements

- [ ] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### PR DESCRIPTION
**Pull request by aws-rafams**
Fixes #657

Also addressing #492 

---
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the project license.

### CODE REVIEW

File: test/cdk-lib/bedrock/integ-tests/kb.integ.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  cdkCommandOptions: {
+    destroy: {
+      args: {
+        force: true,
+      },
+    },
+  },
+});
+
+// const apiCall =
```
Comment (krokoko) [inline test/cdk-lib/bedrock/integ-tests/kb.integ.ts:49]: we should remove or implement the commented code

**Review by krokoko — DISMISSED** [review]

**Comment by krokoko** [comment]
Failing build

**Comment by aws-rafams** [comment]
> Failing build



> Failing build

I see it fails because of an assert on the integ test snapshot. Same reason why I had to comment out part of the code. Will look into it.

**Comment by aws-rafams** [comment]
seems the assert fails due to some non-deterministic operation and my local build succeeds due to some local caching, will continue to look into a possible solution

**Review by krokoko — CHANGES_REQUESTED** [review]
Waiting for https://github.com/awslabs/generative-ai-cdk-constructs/pull/663 to go in before merging this

**Review by krokoko — APPROVED** [review]

**Review by scottschreckengaust — APPROVED** [review]
LGTM

**Pull request merged**